### PR TITLE
doc: add doc for fuse clean policy

### PR DESCRIPTION
**Pull request by BillyChen1**
### Ⅰ. Describe what this PR does
 add doc for fuse clean policy

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/en/samples/fuse_clean_policy.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Before running this example, please refer to the [Installation Documentation](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/userguide/install.md) to complete the installation, and check that each Fluid component is normal run:
+
+```shell
+$ kubectl get pod -n fluid-system
+NAME                                        READY   STATUS    RESTARTS        AGE
+alluxioruntime-controller-86ddc878d-pc6g5   1/1     Running   7 (2m19s ago)   24h
+csi-nodeplugin-fluid-ccbk8                  2/2     Running   6 (2m19s ago)   24h
+dataset-controller-67bcb77b89-6xw7p         1/1     Running   4 (2m16s ago)   24h
+fluid-webhook-648ccc89c6-bq5rd              1/1     Running   4 (2m18s ago)   24h
+```
```
Comment (cheyang) [inline docs/en/samples/fuse_clean_policy.md:17]: Please check the reason of restarting. And I hope the count of restart times is 0. And in the latest version of fluid, alluxioruntime-controller will not be created by default.

File: docs/en/samples/fuse_clean_policy.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+**Check `DataSet` and `AlluxioRuntime` Objects to be Created**
+```shell
+$ cat dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: hbase
+spec:
+  mounts:
+    - mountPoint: https://mirrors.bit.edu.cn/apache/hbase/stable/
```
Comment (cheyang) [inline docs/en/samples/fuse_clean_policy.md:32]: According to 问骁， there are some potential issues when using url `https://mirrors.bit.edu.cn/apache/hbase/stable/`.

File: docs/en/samples/fuse_clean_policy.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# Demo - Set FUSE clean policy
+
+FUSE clean policy is set under the property `spec.fuse.cleanPolicy` of `Runtime`. There are 2 clean policies for FUSE Pod. `OnRuntimeDeleted` means that the FUSE Pod is cleaned only when the cache runtime is deleted, and `OnDemand` means that when the FUSE Pod is not used by any application Pod, the FUSE Pod will be cleaned.
+By default, the cleanup policy for FUSE Pod is `OnRuntimeDeleted`
```
Comment (cheyang) [inline docs/en/samples/fuse_clean_policy.md:4]: If the cleanPolicy is `OnRuntimeDeleted`, how does it work?

Comment (BillyChen1) [reply docs/en/samples/fuse_clean_policy.md:4]: Do you mean add an demo of `OnRuntimeDeleted` showing how it workd?

File: docs/en/samples/fuse_clean_policy.md (hunk 2 above)
Comment (BillyChen1) [reply docs/en/samples/fuse_clean_policy.md:32]: I tested it on ACK just now, it worked.

File: docs/en/samples/fuse_clean_policy.md (hunk 3 above)
Comment (cheyang) [reply docs/en/samples/fuse_clean_policy.md:4]: Yes， I want to make the demo complete.

File: docs/en/samples/fuse_clean_policy.md (hunk 1 above)
Comment (BillyChen1) [reply docs/en/samples/fuse_clean_policy.md:19]: The reason is that I ran Fluid on single-node Kubernetes with my laptop. They would restart when I restarted Kubernetes. 
I've ran the latest version of Fluid on ACK, and everything seemed normal.

**Comment by cheyang** [comment]
@BillyChen1 Could you please investigate why the build is failed. Thanks.

**Comment by BillyChen1** [comment]
> @BillyChen1 Could you please investigate why the build is failed. Thanks.

The reason is that I didn't `make build` locally to update openapi.

**Comment by cheyang** [comment]
@BillyChen1 Please check again.

**Comment by BillyChen1** [comment]
> @BillyChen1 Please check again.

The build was successful, and I just resolved the conflict.

**Comment by BillyChen1** [comment]
@cheyang  Sorry to bother you. I ran `make update-crd` to update the description of `cleanPolicy` in CRDs, now the lint should be going well.

**Comment by cheyang** [comment]
thanks, @BillyChen1

**Review by cheyang — APPROVED** [review]

**Pull request merged**